rskj-core/src/test/java/org/ethereum/rpc/Web3ImplLogsTest.java

```diff
@@ -99,6 +102,7 @@
     public void setUp() {
         RskTestFactory factory = new RskTestFactory();
```

Comment:
Tincho: RskTestFactory is deprecated use  RskTestContext instead which builds the same graph of dependencies than the productive code

Reply:
I did not change that line while implementing this feature.I consider this is out of the scope of this PR.
Still I quick tried your suggestion and it did not work. That means that `RskTestFactory` can not be replaced that easily in this particular case. 
I agree it would be great to have that changed, let's do that someday on a new branch :)